src/main/java/cr0s/warpdrive/block/TileEntityAirGenerator.java

```diff
@@ -111,4 +121,39 @@ public String toString() {
 		worldObj == null ? "~NULL~" : worldObj.getWorldInfo().getWorldName(),
 		xCoord, yCoord, zCoord);
 	}
+	
+	public Object[] enabled(Object[] arguments) {
+	
+		try {
+			if (arguments.length == 1) {
+				boolean newEnable = toBool(arguments[0]);
+				generatorEnabled = newEnable;
+			}
+		} catch (Exception exception) {
```

Comment:
Superfluous try/catch here. The toBool can't fails since arguments length is already tested ahead.